docker/dockerfiles/hl-node.Dockerfile

```diff
@@ -19,53 +19,40 @@ FROM ubuntu:22.04 as node
 
 SHELL ["/bin/bash", "-c"]
 
-# metadata
-ARG VCS_REF
-ARG BUILD_DATE
-ARG IMAGE_NAME
-
 # That can be a single one or a comma separated list
 ARG BINARY=nh-node
-
-ARG BIN_FOLDER=.
-ARG DOC_URL=https://github.com/HorizenLabs/NH-core
-ARG DESCRIPTION="NH-core"
-ARG AUTHORS="devops@horizenlabs.io"
+ARG DESCRIPTION="New Horizen Core"
+ARG AUTHORS="mainchain-team@horizenlabs.io"
 ARG VENDOR="Horizen Labs"
 
-LABEL io.hl.image.authors=${AUTHORS} \
```

Comment:
I agree on removing label `revision`, not sure on removing `created` and `documentation`.

Reply:
My approach was to reduce as many moving parts as possible, so we are not going to be dependent on git repo name and etc in case it changes one day. AFAIK labels are not necessary, but I am ok to keep whatever you guys think MUST be kept. Let me know what do you guys think. FYI: we are not creating any labels like this for either zend or eon IIRC

cc: @lander86